Microsoft.Toolkit.Uwp.UI.Controls.Core/Microsoft.Toolkit.Uwp.UI.Controls.Core.csproj

```diff
@@ -2,9 +2,13 @@
 
```

Comment:
Hmm, was just doing a final check of binary size before merging, and noticed that the `Core` package was smaller 🎉 for some reason with it's Min Footprint (Max footprint has gone down in a lot of cases, but mins are mostly the same):
```
Microsoft.Toolkit.Uwp.UI.Controls.Core Additional Max Footprint: 1,038,662 bytes
Microsoft.Toolkit.Uwp.UI.Controls.Core Additional Min Footprint: 4,204,324 bytes
  App Diff: (.exe) = 6,144
  App Diff: (.xml) = 126
  App Diff: (.appxsym) = 141,569
  Size Diff: AppxBundleManifest.xml = 28
  Size Diff: AppxBlockMap.xml = 1,910
  Size Diff: AppxManifest.xml = 203
  Size Diff: resources.pri = 63,432
  Lib (self): Microsoft.Toolkit.Uwp.UI.Controls.Core.xr.xml = 22,562
  Additional: Microsoft.UI.Xaml.winmd = 249,728
  Additional: Microsoft.Toolkit.dll = 30,720
  Additional: Microsoft.Toolkit.Uwp.dll = 104,448
  Lib (self): Microsoft.Toolkit.Uwp.UI.Controls.Core.dll = 206,336
  Additional: Microsoft.Toolkit.Uwp.UI.Controls.Primitives.dll = 53,760
  Additional: Microsoft.Toolkit.Uwp.UI.dll = 157,696
-----------------COMPILED----------------
  App Diff: (.dll) = 4,135,936
  App Diff: (.exe) = 0
  Size Diff: AppxBundleManifest.xml = 28
  Size Diff: AppxBlockMap.xml = 4,725
  Size Diff: AppxManifest.xml = 203
  Size Diff: resources.pri = 63,432
-----------------------------------------
```

Compared to main:
```
Microsoft.Toolkit.Uwp.UI.Controls.Core Additional Max Footprint: 1,081,383 bytes
Microsoft.Toolkit.Uwp.UI.Controls.Core Additional Min Footprint: 4,922,275 bytes
  App Diff: (.exe) = 6,144
  App Diff: (.xml) = 126
  App Diff: (.appxsym) = 141,453
  Size Diff: AppxBundleManifest.xml = 28
  Size Diff: AppxBlockMap.xml = 1,899
  Size Diff: AppxManifest.xml = 27
  Size Diff: resources.pri = 63,432
  Lib (self): Microsoft.Toolkit.Uwp.UI.Controls.Core.xr.xml = 22,562
  Additional: Microsoft.Toolkit.dll = 39,120
  Additional: Microsoft.Toolkit.Uwp.dll = 112,848
  Lib (self): Microsoft.Toolkit.Uwp.UI.Controls.Core.dll = 214,736
  Additional: Microsoft.Toolkit.Uwp.UI.Controls.Primitives.dll = 62,672
  Additional: Microsoft.Toolkit.Uwp.UI.dll = 166,608
  Additional: Microsoft.UI.Xaml.winmd = 249,728
-----------------COMPILED----------------
  App Diff: (.dll) = 4,603,392
  App Diff: (.exe) = 0
  Size Diff: AppxBundleManifest.xml = 28
  Size Diff: AppxBlockMap.xml = 5,600
  Size Diff: AppxManifest.xml = 27
  Size Diff: resources.pri = 63,432
  Size Diff: [Content_Types].xml = 68
  Additional: Microsoft.UI.Xaml.winmd = 249,728
-----------------------------------------
```

Considering everything should be functionally equivalent this was a surprise. Would be interesting to know what might have changed this. We'll see if we get any reports after of any changes of behavior.

Reply:
I just reviewed my changes; I didn't notice any changes in the project evaluation due to refactor I did in the project files. This is good if there's nothing went wrong with the refactor but let me test the binaries and get back to you.

Reply:
There's CI running on `main` too. Let's see if it's same as the PR.